templates/sag-db-oracle/README.md

```diff
@@ -21,24 +21,42 @@
 
 With this template you can create user, database, and webMethods database schemas on Oracle. By default, the template installs the Database Component Configurator and the database scripts for all products, and it creates database schemas for all database components.
 
-You can remove one or more product database scripts from the template if you do not require to install them. If you remove a database script from the product list, you should also set the components for this database script to empty list. The following table maps the database scripts  to their product and database components: 
-
-Database Script  |  Product | Database Component Property ID
---------------------|----------|---------------------------------------
-OBEcdc              | Optimize |                         db.OBE.components
-WOKcdc              | BusinessRules |                    db.WOK.components
-TNScdc              | TradingNetworks |                  db.TNS.components
-MWScdc              | MyWebmethodsServer |               db.MWS.components
-B2BcloudCdc         | B2B Cloud |                        db.B2Bcloud.components
-MATcdc              | Active Transfer |                  db.MAT.components
-ODEcdc              | One Data |                         db.ODE.components
-PIEcdc              | Integration Server and MSC |       db.PIE.components
-PIEEmbeddedCdc      | Integration Server |               -
-PIEMobileCdc        | Integration Server Mobile Support |db.PIEMobile.components
-WMNcdc              | Monitor |                          db.WMN.components
-WPEcdc              | Process Engine |                   db.WPE.components
-WSTcdc              | Cloud Streams |                    db.WST.components
-YAIcdc              | API Gateway |                      db.YAI.components
+If you do not require to install database schemas for all products, remove the database scripts and the respective database components for the unrequired products. The following table maps the products to their database scripts, database component lists, and database actions for creating schemas:
+
+Product | Database Script | Database Component List | Database Action for Creating Schemas
+--------------------|----------|---------------------|------------------
+webMethods Optimize  |OBEcdc  |  db.OBE.components | schemas.OBE
+webMethods Business Rules | WOKcdc|  db.WOK.components | schemas.WOK
+webMethods Trading Networks | TNScdc  |  db.TNS.components | schemas.TNS
+My webMethods Server | MWScdc| db.MWS.components | schemas.MWS
+webMethods B2B Cloud  |  B2BcloudCdc| db.B2Bcloud.components | schemas.B2Bcloud
+webMethods Active Transfer | MATcdc | db.MAT.components | schemas.MAT
+webMethods OneData |  ODEcdc | db.ODE.components | schemas.ODE
+webMethods Integration Server and webMethods Microservices Runtime | PIEcdc | db.PIE.components | schemas.PIE
+Integration Server | PIEEmbeddedCdc | Not applicable (N/A) | N/A
```

Comment:
PIEEmbeddedCDC is special case - it has no component. i'm not sure if "not applicable" is the correct statement.

Reply:
How about not available?

Reply:
yes. it is good

Reply:
okay, I edited the file